app/assets/stylesheets/homepage.css.scss

```diff
@@ -0,0 +1,6 @@
+.header {
+	width: 100%;
```

Comment:
This was all deleted when we deleted it, are you looking at an older push up? Check out the latest push up requests